data/api/client-server/keys.yaml

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 openapi: 3.1.0
 info:
-  title: Matrix Client-Server Client Config API
+  title: Matrix Client-Server Account Data API
```

Comment:
I don't think this is quite right (but it was already wrong before). Looking at the spec it looks like it should be "Key Management API" for this file.